src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static final int MAX_NUMBER = 100;
+    public static final ColorSuplier colorSuplier = new ColorSuplier();
+    private static Random random = null;
+
+    public static Ball getRandomBall() {
+        random = new Random();
```

Comment:
let's initialize it in fields instead of method

Reply:
https://github.com/mate-academy/jv-lottery/pull/1487#discussion_r1592592318

Reply:
What about it??

Reply:
@[MentorMA](https://github.com/MentorMA) MentorMA [5 hours ago](https://github.com/mate-academy/jv-lottery/pull/1487#discussion_r1592592318)
every invoke will create new object of Randon, create it in field and reuse

Reply:
that's ok :) mentors advised you the same thing